quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/Environment.java

```diff
@@ -194,4 +189,8 @@ public boolean accept(File dir, String name) {
             }
         })).collect(Collectors.toMap(File::getName, Function.identity()));
     }
+
+    public static boolean isQuarkusDevMode() {
```

Comment:
just a note: we now have "isDevMode", "isQuarkusDevMode", "isDevProfile", all for different use cases. May be at least document(javadoc) where we intend to use which one to prevent confusion? I am not absolutely sure if we can't merge some of them, though.

Reply:
We can not. We can document for sure, I thought the names were clear enough though. Will consider that in the next round of changes.